### PR TITLE
Issue 353

### PR DESCRIPTION
**Pull request by mortenjc**
### Issue reference / description

The branch you merge from should already reference an event-formation-unit github ticket number. You can add a descriptive title, but if an issue is referenced, you don't have to.

## Checklist for submitter

- [x] Check for conflict with integration test
- [x] Unit tests pass

---

## Nominate for Group Code Review (Anyone can nominate it)
Indicate if you think the code should be reviewed in a Thursday code review session.

- [ ] Recommend for group code review

Also, nominate it on the code_review Slack channel.

### CODE REVIEW

File: src/efu/Parser.cpp [hunk 1]
```diff
@@ -198,6 +214,12 @@ Parser::Parser(std::shared_ptr<Detector> detector, Statistics &mainStats, int &k
                 return detector_info_get(cmd, resp, nrChars, detector);
               });
 
+
+  registercmd("RUNTIMESTATS", [detector, mainStats](std::vector<std::string> cmd,
```
Comment (amues) [inline src/efu/Parser.cpp:218]: Is `mainStats` required in the capture?

**Review by amues — APPROVED** [review]
Please see the comments in _src/efu/Parser.cpp_ and _src/efu/test/RunTimeStatTest.cpp_.

File: src/common/RuntimeStat.h [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+     OldStats(Counters) {
+       if (Counters.size() == 0) {
+         throw std::runtime_error("RuntimeStat array must have nonzero size");
+       }
+  }
+
+  /// \brief Get  vector of current counters and compare with old values
+  /// if the values change the 'stage' is deemed 'up' and the corresponding
+  /// bit is set.
+  uint32_t getRuntimeStatus(std::vector<int64_t> Stats) {
```
Comment (MortenHilkerSkaaning) [inline src/common/RuntimeStat.h:38]: Do we run this in a hot loop? This will create a copy of `Stats` at every call..

**Comment by mortenjc** [comment]
No, It is run once every second, so I guess it is ok

**Pull request merged**